index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://harshita-crypto-ui.netlify.app/style.css">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <script src="https://kit.fontawesome.com/7fa15c8e24.js" crossorigin="anonymous"></script>
+    <title>Crypto Quiz</title>
+</head>
+<body>
+    <div class="pagewrap">
+        <section class="p-m m-m flex flex-row section">
+            <div class="img-container p-s m-s br-m">
+                <img src="../assets/quiz_home.svg" alt="home quiz svg" class="img-responsive">
+            </div>
+            <div class="home-container p-s m-s">
+                <div class="action-color p-m m-m m-auto txt-xl fw-bold">Welcome to Crypto Quiz</div>
+                <label for="Name" class="p-m m-xs txt-m fw-semibold">Enter your Name:</label>
+                <div class="input-icon relative m-s">
+                    <i class="fas fa-user user-icon"></i>
+                    <div class="flex flex-column">
+                        <input type="text" name="Name" class="br-m p-s txt-s m-s txt-center border-none outline-transparent name fw-semibold">
+                        <a href="./screens/category.html">
+                            <input type="button" value="Play!" class="btn txt-cursor outline-transparent border-none fw-semibold">
```

Comment:
Any specific reason to use a input for button

Reply:
I want to use submit feature but without javascript, it was refreshing on the click so changed it to the button type.